File: src/main/kotlin/com/miro/miroappoauth/dto/UserDto.java
package com.miro.miroappoauth.dto;

public class UserDto {
    private long id;
    private String name;

    public UserDto setId(long id) {
        this.id = id;
        return this;
    }

    public UserDto setName(String name) {
        this.name = name;
        return this;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: src/main/kotlin/com/miro/miroappoauth/dto/UserDto.kt
package com.miro.miroappoauth.dto

import java.time.Instant

data class UserDto(
    // "user", todo enum?
    val type: String,
    val id: Long,
    val name: String,
    val createdAt: Instant,
    val role: String,
    val email: String,
    // "registered", todo enum?
    val state: String
)
